### PR TITLE
Feature/977 add note field to plant details

### PR DESCRIPTION
**Pull request by danielsteinkogler**
## Basics

- [ ] The PR is rebased with current master
- [ ] I added a line to [changelog.md](/doc/changelog.md)
- [ ] Details of what I changed are in the commit messages
- [ ] References to issues, e.g. `close #X`, are in the commit messages and changelog
- [ ] The buildserver is happy

## Checklist

- [ ] I fully described what my PR does in the documentation
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)
- [ ] Exceptions to any guidelines are documented

## First Time Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I am using [Tailwind CSS Linting](https://tailwindcss.com/blog/introducing-linting-for-tailwindcss-intellisense)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] I've read through the whole documentation
- [ ] I've checked conformity to guidelines
- [ ] I've checked conformity to requirements
- [ ] I've checked that the requirements are tested

### CODE REVIEW

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by danielsteinkogler** [comment]
jenkins build please

**Comment by markus2330** [comment]
jenkins build please

File: frontend/src/components/Form/MarkdownEditor.tsx [hunk 1]
```diff
@@ -11,33 +18,104 @@ interface MarkdownEditorProps<T extends FieldValues> {
...
+
+  // custom full screen command to set fullScreen state of the editor
+  const customFullScreenCommand: ICommand = {
+    name: DefaultFullScreenCommand.name,
+    keyCommand: DefaultFullScreenCommand.keyCommand,
+    shortcuts: DefaultFullScreenCommand.shortcuts,
+    value: DefaultFullScreenCommand.value,
+    buttonProps: DefaultFullScreenCommand.buttonProps,
+    icon: DefaultFullScreenCommand.icon,
+    execute: function execute(state, api, dispatch, executeCommandState, shortcuts) {
```
Comment (Bushuo) [inline frontend/src/components/Form/MarkdownEditor.tsx:71]:
```suggestion
    execute(state, api, dispatch, executeCommandState, shortcuts) {
```
Would make it more readable IMO

File: frontend/src/features/map_planning/store/UntrackedMapStore.ts [hunk 2]
```diff
@@ -38,6 +38,15 @@ export const createUntrackedMapSlice: StateCreator<
       y2: 0,
     },
   },
+  setPlantingDetailNotesFullscreen(isFullscreen: boolean) {
```
Comment (Bushuo) [inline frontend/src/features/map_planning/store/UntrackedMapStore.ts:41]: Is this unused?

**Review by Bushuo — CHANGES_REQUESTED** [review]
Sorry reviewing took so long. Please check my remarks.

**Review by Bushuo — APPROVED** [review]

**Comment by markus2330** [comment]
Fantastic work, thank you! :heart:

**Pull request merged**